### DISCUSSION TITLE
Changing card description of page overview via the _category.json

### DISCUSSION THREAD

**Discussion by JPeer264** [facebook/docusaurus#7985] [Ideas]
Docusaurus has a pretty nice feature of having overview pages with all subpages: https://docusaurus.io/docs/category/guides

The good thing is that it displays the description if one card leads to one page. But if the card points to a set of pages it shows "N items". It is actually translatable according to [here](https://github.com/facebook/docusaurus/blob/f683589/packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx#L86-L93) but is very inconvenient if there should be unique descriptions.

It would be nice if there is a way to have this translation only if the `_category.json` does not have a specific entry, such as `description` or similar.

What do you think about that?

**Comment by slorber** [comment]
Yes we want this.
I plan to refactor a bit this category index page in the future and will provide this feature

Also related to: https://github.com/facebook/docusaurus/issues/7818

**Reply by JPeer264** [reply]
Awesome thanks
